webapp/components/ContributionForm/ContributionForm.spec.js

```diff
@@ -228,6 +242,17 @@ describe('ContributionForm.vue', () => {
         await wrapper.find('form').trigger('submit')
         expect(mocks.$apollo.mutate).toHaveBeenCalledWith(expect.objectContaining(expectedParams))
       })
+
+      it('supports updating categories', async () => {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-02T12:14:15Z" title="Tuesday, July 2nd 2019, 2:14:15 pm +02:00">Jul 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -228,6 +242,17 @@ describe('ContributionForm.vue', () => {
         await wrapper.find('form').trigger('submit')
         expect(mocks.$apollo.mutate).toHaveBeenCalledWith(expect.objectContaining(expectedParams))
       })
+
+      it('supports updateing categories', async () => {
```


```suggestion
      it('supports updating categories', async () => {
```